src/operations/find.ts

```diff
@@ -52,7 +52,7 @@ export interface FindOptions extends CommandOperationOptions {
   noCursorTimeout?: boolean;
   /** Specify collation (MongoDB 3.4 or higher) settings for update operation (see 3.4 documentation for available fields). */
   collation?: CollationOptions;
-  /** Enables writing to temporary files on the server. */
+  /** Allows disk use for blocking sort operations exceeding 100MB memory. */
```

Comment:
I wonder if this might be a good opportunity to experiment with improving these docs even more by linking to them: https://docs.mongodb.com/manual/reference/command/find/#find-cmd-allowdiskuse

Also, does typedoc support any way of marking an option supported since a certain MongoDB version? It would be cool to use jsdoc's `@since` tag here

Reply:
`@since` is not supported by tsdoc unfortunately, but we could maybe use [`@remarks`](https://tsdoc.org/pages/tags/remarks/) to separate summary/details? E.g.

```
/**
 * Allows disk use for blocking sort operations exceeding 100MB memory.
 * @remarks
 * Since MongoDB 3.2.
 * {@link https://docs.mongodb.com/manual/reference/command/find/#find-cmd-allowdiskuse | find command allowDiskUse documentation}
 * /
```